site/app/templates/LateDaysTablePlugin.twig

```diff
@@ -27,6 +27,7 @@
             <th>Assignment submitted # of days after deadline</th>
             <th>Student granted # of days extension for this assignment</th>
             <th>Status</th>
+            <th>Grade Inquiries</th>
```

Comment:
It is better to only show this column when grade inquiry is enabled in the course.

Reply:
Will do!